patches/chromium/default_v11_linux_encryption_keys.patch

```diff
@@ -0,0 +1,55 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
I have many questions around this behavior, how it impacts things like cookie encryption, and why we need to change this default at all.

What negative impact does leaving the default as v11 have?

Reply:
One of the reasons for the patch was to bring the linux behaviour in line with MacOS and Windows, with one of electron's benefits being that users can code on one platform and have the code work on all the other main operating systems, I thought that this change made some sense:

- if a user encrypts a string (v11) on linux, stores it somewhere and tries to decrypt on another platform, they would get an error
- linux users are still able to encrypt keys the same way windows and mac users can (v10)

That being said, if you think that this was an awful idea (understandable, I'm a bit shaky on it too) I'm super happy to try and make v11 work out, one key challenge being that assuming CI machines don't have an OS-level library ie. libsecret, I'm not sure how to validate that v11 encryption/decryption works on a linux machine on CI

Reply:
> if a user encrypts a string (v11) on linux, stores it somewhere and tries to decrypt on another platform, they would get an error

A string is encrypted such that moving the string to another platform / device would make it impossible to decrypt.  We should figure out v11